nix/.stack.nix/ouroboros-consensus.nix

```diff
@@ -1,6 +1,6 @@
 { system, compiler, flags, pkgs, hsPkgs, pkgconfPkgs, ... }:
   {
-    flags = {};
+    flags = { checktvarinvariant = false; };
```

Comment:
@edsko do we want this turned on in CI?

Reply:
Probably not by default. Maybe for nightlies.